news/2020-10-17-new-featured-artist-aleph.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "New Featured Artist: ALEPH"
+date: 2020-10-17 09:00:00 +0000
+---
+
+We're proud to welcome **ALEPH** aboard as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/107/header.jpg?20201017)
+
+A musical project seeking to blur the line between clarity and noise, **ALEPH** is an electronic music experience that we've had waiting in the wings since almost the beginning of the Featured Artist initiative. Pieces from the project have been featured on labels like *MAD ZOO*, *Renraku* and *Inspected*, even making appearances on *NEST HQ* and *BBC Radio*, and drawing the support and approval from big names in the scene like *Noisia*, *Mat Zo*, *Amon Tobin*, *The Glitch Mob* and a whole load more.
+
+If you're a fan of electronic music exploration featuring bleeps, boops and absolutely *filthy* bass, **ALEPH** is sure to impress, and we've got **8** new tracks for you to experience, including the rather aptly titled album *BREAKING AND ENTERING*.
+
+On top of that, we've also got two new osu!taiko showcase maps from the eternally-talented Mappers' Guild to help welcome **ALEPH** in to the osu! community.
```

Comment:
> two new osu!taiko showcase maps

one isn't taiko and the other is a year old :(

Reply:
me fix